content/views.py

```diff
@@ -18,11 +17,16 @@
     ArticleSerializer,
     ArticleContentSerializer,
 )
+from common.permissions import (
+    IsCopyeditorOrAbove,
+    IsEditor,
+    create_permission,
+    IsAuthenticated,
+)
```

Comment:
nit: create_permission being in the middle of the permission classes is bothering me

Reply:
Fair and understandable: https://github.com/UWmathNEWS/slugline-api/pull/30/commits/20d22e70a53c6d4fa19efc729ed7cd8868253e75